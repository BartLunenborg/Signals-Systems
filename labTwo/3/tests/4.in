5
2: [0,-4]
3: [4,0,1]
4: [-4,0,2,-2]
5: [-5,3,-2,-1,3]
6: [-2,2,3,2,1,2]
100: [4,5,-2,2,15,-13,-3,-22,-1,-16,18,-14,-23,19,1,-4,-17,6,23,16,-22,-9,-11,-8,21,-7,-1,20,12,-23,-1,-6,7,22,21,-3,-16,-4,2,10,7,-2,21,12,-6,-3,-17,2,-20,-17,-7,-17,1,8,0,-2,-22,-24,-5,17,-22,19,-14,12,19,-15,-13,-22,6,14,16,-12,-13,12,-25,7,-13,-15,-16,-8,-5,5,1,22,-10,-22,-3,-7,-18,19,-15,-13,15,-1,-25,9,9,-13,15,-8]
