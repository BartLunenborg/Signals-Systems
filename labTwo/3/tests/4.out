115: [0,2560,-896,-8320,2464,5056,-33792,8480,1816,-6464,14640,43248,-4896,-5576,70408,-36512,-14608,41944,-3056,-23048,15296,-23576,-9232,-6360,-25904,61024,-7824,-9920,49352,-34624,-52040,49952,-25632,-35120,67256,-10168,-49120,3072,-3800,-34504,17672,19632,-12384,13184,-13472,-36528,7776,-22696,-11768,13504,16072,31720,13264,37312,41576,-5448,2992,4512,-22744,22424,30392,-25728,62768,-6752,-32808,44144,-50096,-18848,17152,-4384,6088,38400,-22584,-3360,19416,-57688,41168,19208,-32776,38832,34424,-4568,33664,31184,-4784,-14640,-24264,19096,-23656,-13584,76272,-21912,-19984,83984,-40608,-38968,86696,-24344,-61792,89504,-28416,-51936,51928,-13664,-27312,26960,-2600,-16360,17096,-7520,-1096,3704,-2760,1040,-384]
